src/applications/vre/28-8832/containers/App.jsx

```diff
@@ -1,12 +1,40 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
+import FormFooter from 'platform/forms/components/FormFooter';
+import WizardContainer from './WizardContainer';
+import { WIZARD_STATUS } from '../constants';
 import formConfig from '../config/form';
 
+// Need to set status of whether or not the wizard is complete to local storage
+// Read from local storage to determine if we should render the wizard OR the intro page
+
 export default function App({ location, children }) {
+  const [wizardState, setWizardState] = useState(false);
+  let content;
+
+  useEffect(
+    () => {
+      const wizardStatus = sessionStorage.getItem(WIZARD_STATUS);
+      setWizardState(JSON.parse(wizardStatus));
+    },
+    [setWizardState],
+  );
+
+  if (!wizardState) {
+    content = <WizardContainer />;
+  } else {
+    content = (
+      <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
+        {children}
+      </RoutedSavableApp>
+    );
+  }
+
   return (
-    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      {children}
-    </RoutedSavableApp>
+    <>
+      {content}
+      <FormFooter formConfig={formConfig} />
```

Comment:
Isn't the `<FormFooter />` part of the `children` passed to the `RoutedSavableApp`?

Reply:
🤔 doesn't look like it? I'd expect a double footer if that were the case, but only see one.